parts/linux/cloud-init/artifacts/kubelet.service

```diff
@@ -1,8 +1,9 @@
 [Unit]
 Description=Kubelet
 ConditionPathExists=/usr/local/bin/kubelet
-Wants=network-online.target containerd.service
+Wants=network-online.target
 After=network-online.target containerd.service
+Requires=containerd.service
```

Comment:
It'd be good to see an e2e test that restarts containerd and checks kubelet also restarts. We have a similar test for chrony - where we kill the process and check it restarts.